scraper/src/youtube2zim/youtube.py

```diff
@@ -319,6 +321,128 @@ def skip_outofrange_videos(date_range, item):
     return dt_parser.parse(item["snippet"]["publishedAt"]).date() in date_range
 
 
+def get_user_short_uploads_playlist_id(channel_id):
```

Comment:
Rather than having your own methods to check playlist exist, I think you should directly rely on `get_playlist_json`, this would greatly reduce the amount of code needed. Could directly be called from `extract_playlists_details_from` in fact, we can even slightly change signature of `get_playlist_json` to return `None` when playlist is just not found and raise an exception in `from_id` where we expect the playlist to not be `None`.